views/includes/templates/content/due-diligence.html

```diff
@@ -0,0 +1,76 @@
+<div class="govuk-grid-row">
```

Comment:
The file path seems to have too many levels? Should it be in includes/inputs (see https://github.com/companieshouse/overseas-entities-web/pull/623/files)

Reply:
This path was created and merged with UAR-101. Do we want to change it now?

Reply:
I think so. Seems wrong to have extra levels (template/content).

Reply:
can we put it in `includes/shared-templates` something like that? One level down instead of in `includes/` level?
Because in that level is just the small, generic stuff shared across the app like back link, continue button, cookie banner etc and not sure about mixing all our shared templates in there.

Reply:
Sure I don't mind one level, page-templates?

Reply:
Okay, cool. @corybudd-ch Il update the UAR-101 content links